src/modules/navigation/MainStackNavigator.tsx

```diff
@@ -15,10 +15,19 @@ import CreateEvent from './CreateEventNavigator';
 import MainTabNavigator from './MainTabNavigator';
 import Notifications from '../notifications/Notifications';
 import {AuthContext} from '../auth/SignUp';
+import { AppStackParamList } from 'App';
 
-const MainStack = createStackNavigator();
+export type MainStackParamList = {
+  JoinGroup: undefined,
```

Comment:
just curious, is there a reason why everything has been given undefined here + the other types you created?

Reply:
Yeah that's supposed to be the type for the route parameters that get passed down to each of the screens (I think it's similar to query parameters on the url for a webpage, but don't quote me on that). These screens don't use any route params so its just undefined.